components/Layouts/Home/HomeNavbar/HomeNavbar.tsx

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { MenuOutlined } from '@ant-design/icons';
+import {
+  BtnSideMenu,
+  Logo,
+  Nav,
+} from 'Components/Layouts/Home/HomeNavbar/HomeNavbar.styled';
+import RightNav from 'Components/Layouts/Home/HomeNavbar/RightNav';
+import SideMenu from 'Components/Layouts/Home/HomeNavbar/SideMenu';
+import React, { useState } from 'react';
+
+const HomeNavbar: React.FC = () => {
+  const [visible, setVisible] = useState(false);
```

Comment:
should able to init `visible` value from props to
```ts
const [visible, setVisible] = useState(props.visible || false)
```

Reply:
Fix here 64a56c474f0bbdadcc9d2498274f69ab1268e332